spreadsheets/introduction/introduction/spreadsheets-features-ii.md

```diff
@@ -27,11 +27,10 @@ This is just some of what spreadsheets can do. We'll cover all the main features
 ---
 ## Practice
 
-What can you **NOT** do using spreadsheets?
+How do you visually represent data in spreadsheets?
 
 ???
 
-* Edit a picture.
-* Calculate numerical data.
-* Create graphs and charts.
-* Create pivot tables.
+* Using graphs or charts.
+* Using images.

```

Comment:
you can also insert images, so this answer could be confusing to some users. how about something like this:

You can use ??? to visually represent data. If the data set is very large, you can use ??? to aggregate the results in an improved view.

* graphs or charts
* pivot tables
* wrong answer 1
* wrong answer 2